pkg/operator/operator.go

```diff
@@ -71,6 +71,11 @@ const (
 	ImageCollector      = "gke.gcr.io/prometheus-engine/prometheus:v2.28.1-gmp.1-gke.1"
 	ImageConfigReloader = "gke.gcr.io/prometheus-engine/config-reloader:v0.1.1-gke.0"
 	ImageRuleEvaluator  = "gke.gcr.io/prometheus-engine/rule-evaluator:v0.1.1-gke.0"
+
+	// The k8s Application, will be exposed as component name.
+	KubernetesAppName     = "app"
+	CollectorAppName      = "managed_prometheus_collector"
+	RuleEvaluatorAppName  = "managed_promehteus_rule_evaulator"
```

Comment:
And we'll want a `app: managed_prometheus_operator` here https://github.com/GoogleCloudPlatform/prometheus-engine/blob/eec06b4fc3c4bf801ff9cc81665074a1854ef1a8/cmd/operator/deploy/operator/05-deployment.yaml#L28 and here https://github.com/GoogleCloudPlatform/prometheus-engine/blob/eec06b4fc3c4bf801ff9cc81665074a1854ef1a8/cmd/operator/deploy/operator/05-deployment.yaml#L34.

Reply:
done.

Reply:
nit: typo `managed_prometheus_rule_evaluator`

Reply:
done